.changeset/fuzzy-gifts-move.md

```diff
@@ -0,0 +1,5 @@
+---
+"@rocket.chat/meteor": patch
+---
+
+fix: new user not moving from Pending to Deactivated tab in admin UI
```

Comment:
Fixes an error where the user was not moving from Pending to Deactivated tab in the admin UI.